src/board/system76/galp5/board.mk

```diff
@@ -5,6 +5,9 @@ EC=it5570e
 # Enable eSPI
 CFLAGS+=-DEC_ESPI=1
 
+# Use S0ix
+CFLAGS+=-DUSE_S0IX=1
+
```

Comment:
As above, this device will not be using S0ix anymore.

Reply:
Done.